Description: disable distutils clean command provided by upstream
 The distutils clean command provided by upstream is disabled because
 it fails to recognize -a flag. The problem is resolved when that
 command is disabled.
Bug-Debian: http://bugs.debian.org/590633
Forwarded: no
Last-Update: 2010-11-17

--- pyzmq-2.0.10.orig/setup.py
+++ pyzmq-2.0.10/setup.py
@@ -154,7 +154,7 @@ if ignore_common_warnings:
 # Extensions
 #-----------------------------------------------------------------------------
 
-cmdclass = {'test':TestCommand, 'clean':CleanCommand }
+cmdclass = {'test':TestCommand}
 
 includes = [pjoin('zmq', sub) for sub in ('utils','core','devices')]
 
